Common/HHKinFitSelection.py

```diff
@@ -0,0 +1,61 @@
+import ROOT
+import os
+from .Utilities import *
+
+initialized = False
+
+def Initialize():
+    global initialized
+    if initialized:
+        raise RuntimeError('HH KinFitSel already initialized')
+    headers_dir = os.path.dirname(os.path.abspath(__file__))
+    header_path_HHKinFit = os.path.join(headers_dir, "KinFitInterface.h")
+    ROOT.gInterpreter.Declare(f'#include "{header_path_HHKinFit}"')
+    header_path_SVFit = os.path.join(headers_dir, "SVfitAnaInterface.h")
+    ROOT.gInterpreter.Declare(f'#include "{header_path_SVFit}"')
+    header_path_MT2 = os.path.join(headers_dir, "MT2.h")
+    ROOT.gInterpreter.Declare(f'#include "{header_path_MT2}"')
+    header_path_Lester_mt2_bisect = os.path.join(headers_dir, "Lester_mt2_bisect.cpp")
+    ROOT.gInterpreter.Declare(f'#include "{header_path_Lester_mt2_bisect}"')
+    initialized = True
+
+
+def GetKinFitConvergence(df):
```

Comment:
split into `GetKinFit`, `GetSVfit`, and `GetMT2`

Reply:
ok